src/app/__init__.py

```diff
@@ -0,0 +1,47 @@
+from flask import Flask
+from werkzeug.routing import BaseConverter
+from werkzeug.http import HTTP_STATUS_CODES
+from .helpers import get_view_args, chapter_lang_exists, ebook_exists, get_ebook_methodology, \
+    add_footnote_links, year_live, accentless_sort
+from config import TEMPLATES_DIR, STATIC_DIR
+import logging
+
+
+# Set WOFF and WOFF2 caching to return 1 year as they should never change
+# Note this requires similar set up in app.yaml for Google App Engine
+class MyFlask(Flask):
+    def get_send_file_max_age(self, name):
+        if name.lower().endswith('.woff') or name.lower().endswith('.woff2'):
+            return 31536000
+        return Flask.get_send_file_max_age(self, name)
+
+
+class RegexConverter(BaseConverter):
+    def __init__(self, url_map, *items):
+        super(RegexConverter, self).__init__(url_map)
+        self.regex = items[0]
+
+
+def create_app():
```

Comment:
A bit more descriptive
```suggestion
def initialize_server():
```